storm_control/hal4000/camera/noneCameraControl.py

```diff
@@ -135,7 +135,8 @@ def newParameters(self, parameters, initialization = False):
             self.fake_frame = numpy.zeros(size_x * size_y, dtype = numpy.uint16)
             for i in range(size_x):
                 for j in range(size_y):
-                    self.fake_frame[j*size_x+i] = i % 128 + j % 128
+                    #self.fake_frame[j*size_x+i] = i % 128 + j % 128
+                    self.fake_frame[j*size_x+i] = 0

```

Comment:
NIT: May just want to add a comment here explaining why we decided to make the background black instead of the rainbow pattern. Just in case anyone else comes looking